Cargo.toml

```diff
@@ -58,7 +58,8 @@ protobuf = "2.20.0"
 rand = "0.8"
 regex = "1.4"
 # pin to 0.19: https://github.com/getsentry/sentry-rust/issues/277
-sentry = { version = "0.19", features = ["with_curl_transport"] }  # pin to 0.19 due to curl transport, failure
+sentry = { version = "0.19", features = ["with_curl_transport"] }# pin to 0.19 due to curl transport, failure
```

Comment:
huh, we can probably drop the `failure` comment thing here, because we no longer need it.